apps/frontend/src/routes/blog.$blogId.lazy.tsx

```
@@ -0,0 +1,81 @@
+                    <figure
+                        className={`w-lvw max-w-prose place-self-center ${!post.image && 'hidden'}`}
+                    >
+                        <img
+                            src={post.image || undefined}
+                            className="w-full object-contain sm:rounded-2xl"
+                        />
+                    </figure>
+
+                    <section
+                        className="2xs:text-base 2xs:leading-7 flex flex-col gap-2 text-sm leading-7"
+                        dangerouslySetInnerHTML={{ __html: post.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)